LSP-tailwindcss.sublime-settings

```diff
@@ -1,92 +1,81 @@
 {
-	// if you are developing this plugin...
-	"developing": false,
-	//////////////////
-	// LSP settings //
-	//////////////////
 	"languages": [
```

Comment:
Why not replace this entire `languages` array with a `selector`?

Reply:
is selector supported is ST 3?

Reply:
Why would you support anything other than ST4?

Reply:
We haven't really discussed dropping support for ST 3. 
So I thought I would keep it, as `lsp_utils` gives me backwards compatibility with ST3.

Reply:
I created an discussion at the lsp repo https://github.com/sublimelsp/LSP/issues/1697